nextjs-13/src/components/CreateNewsletterModal/index.test.tsx

```diff
@@ -69,7 +81,11 @@ describe('CreateNewsletterModal', () => {
 
   it('enters name', () => {
```

Comment:
Reference: 
https://www.betterspecs.org/#let
https://www.betterspecs.org/#should

Reply:
I understand your first comment on this issue, but don't, with the second one 🤣

Jest not having anything like `let`, right? 👀 

And about the `should`, do you means that we should rename the `it('enters name')` into something better? I agree, since `enters name` seems somewhat confusing.

Reply:
for this first I mean the `it('enters name')` and `it('enters content')` are doing the same things, but now I just realize that you have an input element on `enters name` and textarea element on `enters contents` both are a different element, then the way you have done it should good. 

and Yes, you should rename the `it ('enters name')` to something better like `sets the name field`

Sorry, for making you confuse here 🙏🏼

Reply:
Understood! Thanks for the clarification.

Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/4c2568f1e4e90ec70f664d467ac38f4f5bc91557) ✋